docs_user/modules/proc_adopting-the-networking-service.adoc

```diff
@@ -2,32 +2,31 @@
 
 = Adopting the {networking_service}
 
-Adopting {networking_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where the {networking_service}
-is supposed to be disabled, should be patched to start the service with the
-configuration parameters provided by the source environment.
+To adopt the {networking_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR), where the {networking_service}
+is disabled. The patch starts the service with the
+configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
 
-When the procedure is over, the expectation is to see the `NeutronAPI` service is running: the {identity_service_first_ref} endpoints should be updated and the same backend of the source Cloud will be available. If the conditions above are met, the adoption is considered concluded.
+The {networking_service} adoption is complete if you see the following results:
 
-This guide also assumes that:
-
-. A {OpenStackPreviousInstaller} environment (the source Cloud) is running on one side;
-. A `SNO` / `CodeReadyContainers` is running on the other side.
+* The `NeutronAPI` service is running.
+* The {identity_service_first_ref} endpoints are updated, and the same back end of the source cloud is available.
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, MariaDB,{identity_service_first_ref}, and xref:migrating-ovn-data_migrating-databases[Migrating OVN data]
-should be already adopted.
+* Ensure that Single Node OpenShift or OpenShift Local is running in the {OpenShift} cluster.
+* Adopt the {identity_service}. For more information, see xref:adopting-the-identity-service_adopt-control-plane[Adopting the {identity_service}].
+* Migrate your OVN databases to `ovsdb-server` instances that run in the {OpenShift} cluster. For more information, see xref:migrating-ovn-data_migrating-databases[Migrating OVN data].
+
 
 .Procedure
-//The following link takes me to a 404. Do we need this text? I think we should start the procedure at "Patch OpenStackControlPlane..."
 ifeval::["{build}" != "downstream"]
 As already done for https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone], the Neutron Adoption follows the same pattern.
```

Comment:
Maybe "The Neutron Adoption follows a similar pattern to https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone]"